algorithms/DEADataHandling.py

```diff
@@ -88,3 +91,52 @@ def load_nbarx(sensor, query, bands_of_interest, product = 'nbart'):
         return ds, crs, affine
     else:
         return None
+
+def load_sentinel(dc, product, query, **bands_of_interest): 
+    '''loads a sentinel granule product and masks using pq
+    Last modified: March 2018
+    Claire Krause: Bex Dunn
+    
+    This function requires the following be loaded:
+    from datacube.helpers import ga_pq_fuser
+    from datacube.storage import masking
+    from datacube import Datacube
+    
+    inputs
+    dc - handle for the Datacube to import from. This allows you to also use dev environments
+	 if that have been imported into the environment.
+    product - string containing the name of the sentinel product to load
+    query - A dict containing the query bounds. Can include lat/lon, time etc
+
+    optional:
+    bands_of_interest - List of strings containing the bands to be read in. Options
+                       'red', 'green', 'blue', 'nir', 'swir1', 'swir2'
```

Comment:
These aren't the only sentinel bands... could update with full list of bands tabled in "GettingStartedWithSentinel2", or direct users to call "dc.list_measurements()" given there's many of them

Reply:
Whoops, didn't realise leaving a comment would prevent me from accepting this... I think @BexDunn should still be able to close it!